maven-plugin-report-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginReportNoFork.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.plugin.plugin.report;
+
+import org.apache.maven.plugins.annotations.Execute;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+
+/**
+ * Generates the Plugin's documentation report: <code>plugin-info.html</code> plugin overview page,
+ * and one <code><i>goal</i>-mojo.html</code> per goal.
+ * Relies on one output file from <a href="../maven-plugin-plugin/descriptor-mojo.html">plugin:descriptor</a>.
+ *
+ * @since 3.14.0
```

Comment:
3.x is currently 3.13.2, but I would expect a new goal to be first include in a minor version instead.

https://github.com/apache/maven-plugin-tools/blob/ad93285f66b95cfe186ee83f53e4e520241eee35/pom.xml#L32

Reply:
Correct, that is also what I would do. Bump minor version.

Reply:
Should I bump the POM in this PR?

Reply:
As a separate, preluding commit.

Reply:
version bump > https://github.com/apache/maven-plugin-tools/pull/311